projects/m1/010-month-name-to-number-of-days/solutions/jimmyhu/solution-month-name-to-number-of-days.py

```diff
@@ -0,0 +1,14 @@
+long_month = ('january','march','may','july','august','october','dicember')
+medium_month = ('april','june','september','november')
+special_month = 'february'
+mese = input('please insert the month(en): ').lower()
+if mese in long_month:
+    print('this month have 31 days')
+elif mese in medium_month:
+    print('this month have 30 days')
+elif mese == special_month:
```

Comment:
When you fall in a similar situation (you have one single condition that is an exception from the others) I suggest to use it as an exit early condition, in order to avoid your code to be processed uselessly.